core/dragging/dragger.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {IDragger} from '../interfaces/i_dragger.js';
+import {IDraggable} from '../interfaces/i_draggable.js';
+import {Coordinate} from '../utils/coordinate.js';
+import {WorkspaceSvg} from '../workspace_svg.js';
+
+export class Dragger implements IDragger {
+  private startLoc: Coordinate;
```

Comment:
Please note the coordinate system. (It's not hard to figure out, but might as well just note it)